### DISCUSSION TITLE
Question about indirect relations

### DISCUSSION THREAD

**Discussion by aonghas** [pocketbase/pocketbase#5720] [Q&A] [answered]
Hi there,

Just a follow up question in relation to the indirect expansion I found from the thread here (https://github.com/pocketbase/pocketbase/issues/312)

- Will the parent record "listen" to changes within the expanded records and trigger a realtime event as well?
- Is there a way to order the expanded relations by one of its fields (eg. "order") via an API query?

Thanks!
Aonghas

**Comment by ganigeorgiev** [comment, marked as answer]
1. The collection realtime events are triggered only for the specific collection records and not their relations, aka. if you subscribe to `example` collection you'll receive events only for `example` records changes.

2. No. If you need pagination, sorting and filtering of the expanded relations you need to send a separate `getList` request.

**Reply by aonghas** [reply]
Cool thanks, I guess I can do the initial get request with the relations so they can all be in one call, and then subsequently subscribe to the related table with the filtered records for realtime events of those.
